cider/exec/nextgen/jitlib/base/JITValue.h

```diff
@@ -153,6 +153,12 @@ class JITValuePointer {
 
   JITValuePointer operator[](JITValue& index);
 
+  JITValuePointer(JITValue& value) : ptr_(&value) {
+    if (ptr_) {
+      ptr_->addRef();
+    }
+  }
```

Comment:
@bigPYJ1151 Please take a look at this method.

Reply:
This method has been fixed in newest commit of main branch. Some interfaces have changed, please rebase this PR to newest main branch and fix some potential problems, thx~.